src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Android.cs

```diff
@@ -0,0 +1,89 @@
+﻿#if __ANDROID_29__
+using AndroidX.Core.View;
+#else
+using Android.Support.V4.View;
+#endif
+using System;
+using System.ComponentModel;
+using Android.Content;
+using Android.Graphics.Drawables;
+using Android.Runtime;
+using Java.IO;
+using Android.Graphics;
+using Windows.UI.Xaml.Media;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+using Uno.Logging;
+using Uno.UI.Controls;
+using Uno.UI;
+using Android.Views;
+using Windows.UI.Xaml.Controls;
+
+namespace Windows.UI.Xaml.Media
+{
+	/// <summary>
+	/// Renderer to update all frames with better shadows matching material design standards.
+	/// </summary>	
+	public partial class AcrylicBrush
+	{
+		private void UpdateProperties(AcrylicState state)
+		{
+			UpdateCornerRadius();
+			UpdateTint(state);
+		}
+
+		private void UpdateCornerRadius()
+		{
+			//TODO
+			//_acrylicLayer?.SetCornerRadius(ContextHelper.Current.ToPixels(MaterialFrame.CornerRadius));
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)